### PR TITLE
Work in Progress

### PR DESCRIPTION
**Pull request by irobayna**
@pi0 This is my first cut, can you take a look at the plugins/font-awesome.js file and see what else I need ?  

I am not 100% sure  I have implemented the extend function correctly

### CODE REVIEW

File: plugins/font-awesome.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import Vue from 'vue';
+
+export const extend = function (config) {
+  css: [
```
Comment (pi0) [inline plugins/font-awesome.js]: - path to node_modules should be relative to project's rootDir
- extend is just a function which mutates nuxt config.we should push this line to css.
Something like this:
```js
var fa = 'node_modules/font-awesome/css/font-awesome.css';
if (config.rootDir) {
  fa = path.resolve(config.rootDir, fa);
}
config.css.push(fa)
```

**Comment by pi0** [comment]
Thanks @irobayna . Also please revert back yarn.lock file :) It should remains in repository as advised by yarn project.

File: plugins/font-awesome.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import Vue from 'vue';
```
Comment (pi0) [inline plugins/font-awesome.js]: This import is unnecessary

**Comment by irobayna** [comment]
@pi0 Thank you for the help, I have made the two changes now

**Comment by pi0** [comment]
Seems nice :) I'll merge it, however we should test all functionality before release. (Can you test and confirm this using yarn link ? )

**Pull request merged**